cypress/e2e/with-users/machines/list.spec.ts

```diff
@@ -39,6 +39,31 @@ context("Machine listing", () => {
     });
   });
 
+  it("displays machine counts with active filters", () => {
+    const searchFilter = "status:(=commissioning) hostname:(machine-)";
+    cy.addMachines(["machine-1", "machine-2"]);
+    cy.findByRole("combobox", { name: "Group by" }).select("Group by status");
+    cy.findByRole("searchbox").type(searchFilter);
+    cy.findByText(/2 machines available/).should("exist");
+    cy.waitForTableToLoad({ name: "Machines" }).within(() =>
+      // eslint-disable-next-line cypress/no-force
```

Comment:
Does this line still need to be here for this to work?

Reply:
Yes, cypress doesn't like how our checkboxes are styled (the native checkbox is hidden) and recognizes this as a potential issue.

Reply:
Ahhh damn that's fair. Yeah I had some issues with that before